pages/common/clear.md

```diff
@@ -2,6 +2,18 @@
 
 > Clears the screen of the terminal.
 
-- Clear the screen (equivalent to typing Control-L when using the bash shell):
+- Clear the screen (equivalent to pressing Control-L in Bash shell):
 
 `clear`
+
+- Clear the screen but keep the terminal's scrollback buffer:
+
+`clear -x`
+
+- Indicate the type of terminal to clean (normally this option is unnecessary because it is taken from the environment variable `TERM`):
```

Comment:
Perhaps this will read better?
```suggestion
- Indicate the type of terminal to clean (defaults to the value of the environment variable `TERM`):
```